moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -383,6 +412,10 @@ class Robot
   bool isReachable(std::unique_ptr<TrajectoryPoint> & point, double timeout,
                    std::vector<double> joint_seed = std::vector<double>() ) const;
 
+  bool jointTrajectoryCollisionCheck(control_msgs::FollowJointTrajectoryGoal & goal,
```

Comment:
This could be shorter...again the parameters indicate this works on a joint trajectory...so no need for an extra long name.